fr-fr/javascript-fr.html.markdown

```diff
@@ -328,13 +328,14 @@ for (var x in person){
 }
 description; // = "Paul Ken 18 "
 
-// *ES6:* La boucle for...of permet d'itérer sur les propriétés d'un objet
-var description = "";
-var person = {fname:"Paul", lname:"Ken", age:18};
-for (var x of person){
-    description += x + " ";
+// *ES6:* La boucle for...of permet de parcourir un objet itérable
+// (ce qui inclut les objets Array, Map, Set, String, ... Mais pas un objet littéral !)
+var myPets = "";
```

Comment:
let's use "let", var should be avoided in any new code

Reply:
updated at 4c9e1c8